test/channels/post_channel_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class PostChannelTest < ActionCable::Channel::TestCase
+  # test "subscribes" do
+  #   subscribe
+  #   assert subscription.confirmed?
+  # end
+end
```

Comment:
Not strictly necessary. It is a test file but I do not believe you are writing tests for your projects.